docs/development/nodejs/how-to-install-nodejs-and-nginx-on-centos-8/index.md

```diff
@@ -0,0 +1,303 @@
+
+{{< note >}}
+Ensure you replace `example.com` with your own site's name or IP address in all commands and examples in this section.
+{{</ note >}}
+
+1. Create the `test.js` file in your site's root directory.
+
+      {{< file "/var/www/example.com/test.js" >}}
+<!DOCTYPE html>
+<html>
+<body>
+
+<h2>
+Your Node.JS server is working.
+</h2>
+
+<p>
+The below button is technically dynamic. You are now using Javascript on both the client-side and the server-side.</p>
+
+<button type="button" onclick="document.getElementById('sample').innerHTML = Date()"> Display the date and time.</button>
+<p id="sample"></p>
+
+</body>
+</html>
+    {{</ file >}}
+
+### Create Your the Node.js Web Server File
+
+In this section, you will create a file named `server.js` that will use Node.js modules to help you write a simple web server that can handle client requests and return responses to them.
+
+1. In your site's root directory, create the `server.js` file with the following content.
+
+      {{< file "/var/www/example.com/server.js">}}
+//nodejs.org/api for API docs
+//Node.js web server
+var http = require("http"),                           //Import Node.js modules
+    url = require("url"),
+    path = require("path"),
+    fs = require("fs");
+
+http.createServer(function(request, response) {       //Create server
+var name = url.parse(request.url).pathname;           //Parse URL
+var filename = path.join(process.cwd(), name);        //Create filename
+fs.readFile(filename, "binary", function(err, file) { //Read file
+    if(err) {                                         //Tracking Errors
+        response.writeHead(500, {"Content-Type": "text/plain"});
+        response.write(err + "\n");
+        response.end();
+        return;
+    }
+    response.writeHead(200);                          //Header request response
+    response.write(file, "binary");                   //Sends body response
+    response.end();                                   //Signals to server that
+ });                                                  //header and body sent
+}).listen(3000);                                      //Listening port
+console.log("Server is listening on port 3000.")      //Terminal output
+        {{</ file >}}
+
+
+1.  Run a new [tmux](/docs/networking/ssh/persistent-terminal-sessions-with-tmux/) session:
+
+        tmux
+    Press **return** when prompted.
+
+1. Navigate to your root directory where your `test.js` file is located.
+
+        cd /var/www/example.com
+
+1. Run your Node.js web server. Appending `&` to the end of a command will keep the web server's process running in the background.
+
+        node server.js &
+
+    You should see your terminal return a process ID after issuing the previous command. Return to your command prompt by entering **CTRL+C**.
+
+1.  Exit your screen session.
```

Comment:
Is this called a screen or a session in the tmux world?

Reply:
tmux session!